src/main/java/gregtech/api/capability/impl/AbstractRecipeLogic.java

```diff
@@ -405,26 +434,11 @@ protected boolean setupAndConsumeRecipeInputs(Recipe recipe, IItemHandlerModifia
                 capacity = totalEUt;
             }
 
-            enoughPower = getEnergyStored() >= capacity;
+            return getEnergyStored() >= capacity;
         } else {
             int power = resultOverclock[0];
```

Comment:
Generators themselves do not generate 2 amps, but have outputs that can output up to 2 amps. Power generated should remain the same. The only thing that changes is the rate at which it can be extracted.